services/src/atdd-staging/src/main/java/org/openkilda/atdd/staging/steps/helpers/FlowSet.java

```diff
@@ -44,6 +44,19 @@
         return unmodifiableSet(flows);
     }
 
+    /**
+     * Returns an unallocated vlan. The returned vlan is immediately added to the list of allocated vlans
+     */
+    public Integer allocateVlan() {
```

Comment:
Any reason to use wrapper class instead of primitive? If it can be null, then a check is missing.